Api/app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,10 @@
+class NotificationsController < ApplicationController
+  def index
+    render json: { has_notifications: current_user.notify_user.where(status: "Unread").size > 0 }
+  end
+
+  def create
```

Comment:
I strongly think that the creation of a new notification is missing because only there is the update notifications.